### ISSUE TITLE
somministrazioni-vaccini-summary-latest.csv - non cambiate le vecchie posizioni

### ISSUE THREAD

**Issue by lmarceg** [italia/covid19-opendata-vaccini#158]
Ciao,
utilizzo quel file per tenere traccia dell'andamento dei vaccini, ma se il numero relativo alla seconda dose si trovava in colonna 14, adesso ho visto che quel dato è stato spostato in colonna 17.
Per evitare di aggiornare lo script ad ogni cambiamento, troverei piú sensato aggiungere i nuovi dati alla fine del csv e non in mezzo, in modo che i dati vecchi non cambino mai di posto.
Grazie!

**Comment by FrancescoDiMaggio** [comment]
Qui stiamo parlando delle basi proprio... concordo.

**Comment by marcosolina** [comment]
Concordo con Imarceg. Se può essere utile, quando mi sono accorto dello "shift" ho deciso di recuperare i dati in base al nome della colonna anziche usare la posizione. Questo dovrebbe ovviare al problema nel caso in cui in futuro aggiungano altre colonne "in mezzo", a patto che non cambino anche i nomi già esistenti. Qui il mio codice se volete dare un'occhiata https://github.com/marcosolina/covidstatus/blob/main/src/main/java/com/marco/javacovidstatus/services/implementations/VaccinesGivenDownloader.java

**Comment by jenkin** [comment]
In generale, non si deve **MAI** accedere al valore di una cella per posizione, **MAI MAI MAI MAI MAI**. L'unico modo corretto prevede di fare il parsing del CSV e poi accedere per chiave (il nome della colonna, che non dovrebbe mai cambiare).
